projects/isolate_proto/src/isolate_proto/controller.proto

```diff
@@ -108,6 +108,13 @@ message UserKeyInfo {
     string alias = 4;
 }
 
+message ServiceURLs {
+    string playground = 1;
+    string run = 2;
+    string queue = 3;
+    string ws = 4;
+}
```

Comment:
We have a lot of magic string manipulation to figure these out in the client. I think our backend should just send these in itself, similar how our queue client sends response_url/status_url/etc - which is very convenient and removes the quess work completely.